src/ui/ResponsiveComponents/ResponsiveComponentsManager.ts

```diff
@@ -101,16 +110,18 @@ export class ResponsiveComponentsManager {
     this.dropdownHeadersWrapper = $$('div', { className: ResponsiveComponentsManager.DROPDOWN_HEADER_WRAPPER_CSS_CLASS });
     this.searchBoxElement = this.getSearchBoxElement();
     this.resizeListener = _.debounce(() => {
-      this.addDropdownHeaderWrapperIfNeeded();
-      if (this.shouldSwitchToSmallMode()) {
-        this.coveoRoot.addClass('coveo-small-interface');
-      } else if (!this.shouldSwitchToSmallMode()) {
-        this.coveoRoot.removeClass('coveo-small-interface');
+      if (this.coveoRoot.width() != 0) {
```

Comment:
Also, explaining the consequence of this would be useful (explaining that facet won't hide in a menu, tabs, won't resize correctly in a dropdown, etc.)